exercises/practice/two-fer/.meta/config.json

```
@@ -1,6 +1,13 @@
 {
   "blurb": "Create a sentence of the form \"One for X, one for me.\"",
-  "authors": [],
+  "authors": [
+    "amscotti"
+  ],
+  "contributors": [
+    "devkabiir",
+    "ErikSchierboom",
+    "Stargator"
+  ],
```

```suggestion
  "contributors": [
  ],
```

---

I suggest adding my name due to an update that changed the API, the method used by the test suite for running the tests against. I would say that is a substantial change.

```suggestion
  "contributors": [
    "Stargator"
  ],
```